src/reconcile/rec_write.c

```diff
@@ -1676,6 +1676,90 @@ __rec_compression_adjust(WT_SESSION_IMPL *session, uint32_t max, size_t compress
     *adjustp = new;
 }
 
+/*
+ * __rec_page_time_stats --
+ *     Update statistics about this page, don't bother clearing here - they are cleared in a
+ *     different place, because sometimes we accumulate these stats and don't end up writing.
+ */
```

Comment:
This doesn't appear to be correct, the function is only called from one place and could reasonable clear the fields.

Reply:
It's the goto `copy_image` that's important, I think. We want to cleanup regardless of whether we write the page out, but only want to record stats when the page is actually written out.

Reply:
Got it, understood.